tests/performance/tests/testK6Integration.js

```diff
@@ -12,6 +14,8 @@ export const options = defaultOptions;
 
 const runAllGroups = (header, tags) => {
     mainDashboard(__ENV.HOST, header, tags);
+    vulnerabilityManagementDashboard(__ENV.HOST, header, tags);
+    alertsGrpc(__ENV.HOST, header, tags);
```

Comment:
Maybe we should pull `[0, 10, 100, 1000]. forEach` outside of the group into the test.

```
[0, 10, 100, 1000].forEach(limit => { alertsGrpc(__ENV.HOST, header, tags, limit); })
```

All other groups are structured in a similar way.

Reply:
Fixed